lms/envs/production.py

```diff
@@ -536,11 +536,12 @@
 # normally appends to every returned URL.
 AWS_QUERYSTRING_AUTH = AUTH_TOKENS.get('AWS_QUERYSTRING_AUTH', True)
 AWS_S3_CUSTOM_DOMAIN = AUTH_TOKENS.get('AWS_S3_CUSTOM_DOMAIN', 'edxuploads.s3.amazonaws.com')
+AWS_S3_ENCRYPTION = False
```

Comment:
Same here, this variable needs to be configurable from the JSON/YAML files.

Reply:
Done